common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -1726,11 +1727,22 @@
             "@typescript-eslint/parser": [
                 ".ts",
                 ".tsx",
-                ".d.ts"
+                ".d.ts",
+                ".tsx"
             ]
         },
         "import/resolver": {
             "node": {
+                "extensions": [
```

Comment:
Both `node` and `typescript` appear here now as resolvers. Is this correct? And should we _not_ be including `js` modules in the list to be resolved via typescript?

Reply:
I can't explain what's happening here unfortunately. I followed https://github.com/import-js/eslint-import-resolver-typescript#configuration as closely as possible, and the testing worked. But I agree this is weird and I don't know what the "right" settings are beyond what "worked". :(